300-quarkus-vertx-webClient/src/test/java/io/quarkus/qe/vertx/webclient/ChuckNorrisResourceTest.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkus.qe.vertx.webclient;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+@QuarkusTestResource(WireMockChuckNorrisResource.class)
+public class ChuckNorrisResourceTest {
+
+    final String EXPECTED_ID = "aBanNLDwR-SAz7iMHuCiyw";
+    final String EXPECTED_VALUE = "Chuck Norris has already been to mars; that why there's no signs of life";
+
+    @Test
+    @DisplayName("Vert.x WebClient [flavor: mutiny] -> Map json response body to POJO")
+    public void getChuckJokeAsJSON(){
+
+        stubFor(get(urlEqualTo("/jokes/random"))
+                .willReturn(aResponse()
+                        .withHeader("Accept", "application/json")
+                        .withBody(String.format("{\"categories\":[]," +
```

Comment:
This is only of my preference: I would prefer working with classes rather than using plain text as json. I guess we can use the default json marshaller provided by Quarkus, inject it here and then unmarshall it to string.

Reply:
I'm like 50/50 with this. I don't mind keeping it as is.